source/sync/go-to-production/optimize-sync-atlas-usage.txt

```diff
@@ -18,12 +18,12 @@ Optimize Sync Atlas Usage
 Overview
 --------
 
-Atlas Device Sync uses space in your app's synced Atlas cluster to store utility
-data for sync. This includes a history of changes to each realm.
-Atlas App Services minimizes that space usage via backend
-compaction, which runs automatically. You can optimize further by
-specifying a client maximum offline time, which enables advanced backend
-compaction.
+Atlas Device Sync uses space in your app's synced Atlas cluster to store 
+utility data for sync. This includes a history of changes to each realm. 
+Atlas App Services minimizes this space usage in your Atlas cluster via 
+:ref:`Trimming <trimming>` for Flexible Sync apps or 
+:ref:`backend compaction <backend-compaction>` for Partition Based Sync apps.
+Minimizing utility data is necessary to reduce the time and data needed for sync.
```

Comment:
I think we could safely get rid of this last sentence as its not adding too much, but will defer to you